CMakeLists.txt

```
@@ -366,6 +377,17 @@ if (BACKTRACE)
     endif ()
 endif ()
 
+if ((LOCALIZE OR BUILD_TESTING) AND "${GETTEXT_MSGFMT_BINARY}" STREQUAL "")
+    if(MSVC)
+        list(APPEND Gettext_ROOT C:\\msys64\\usr)
+        list(APPEND Gettext_ROOT C:\\Program\ Files\\Git\\mingw64)
```

With this change localization works. However we need to fix the behavior if msgfmt is unavailable.
```suggestion
        list(APPEND Gettext_ROOT C:\\Program\ Files\\Git\\usr)
```

---

Or maybe just have both paths in hints. I'm not sure if the location my file is in is canonical or not.